corehq/apps/export/forms.py

```diff
@@ -184,42 +184,43 @@ def __init__(self, domain_object, *args, **kwargs):
         self.helper.form_tag = False
         self.helper.label_class = 'col-sm-3'
         self.helper.field_class = 'col-sm-5'
-        self.helper.layout = Layout(
-            crispy.Field(
-                'type_or_group',
-                ng_model="formData.type_or_group",
-                ng_required='false',
-            ),
-            crispy.Div(
+        if not (hasattr(self, 'skip_layout') and self.skip_layout):
```

Comment:
It sure is. Nice move.. thats cool!!!

Absolutely!! should add that as a class attribute with default False. Would also help add some documentation for its use.